packages/openneuro-components/src/search-page/SearchPageContainerExample.tsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react'
+import { FacetBlockContainerExample } from './FacetBlockContainerExample'
+import { SearchResults } from './SearchResults'
+import { FiltersBlock } from './FiltersBlock'
+import { SearchPage } from './SearchPage'
+import { SearchSortContainerExample } from './SearchSortContainerExample'
+import { KeywordInputContainerExample } from './KeywordInputContainerExample'
+import { sortBy } from '../mock-content/sortby-list'
+import { FacetSelect } from '../facets/FacetSelect'
+import { FacetRadio } from '../facets/FacetRadio'
+import { FacetRange } from '../facets/FacetRange'
+
+import {
+  modalities,
+  show_available,
+  showMyUploads_available,
+  diagnosis,
+  task,
+  author_pi,
+  gender,
+  species,
+  section,
+  domain,
+} from '../mock-content/facet-content'
+import { AccordionWrap } from '../accordion/AccordionWrap'
+import { AccordionTab } from '../accordion/AccordionTab'
+
+import './search-page.scss'
+
+export interface SearchContainereProps {
+  portalContent?: Record<string, any>
+  searchResults
+  profile?: Record<string, any>
+}
+
+export const SearchPageContainerExample = ({
+  searchResults,
+  portalContent,
+  profile,
+}: SearchContainereProps) => {
+  const [selected, setSelected] = React.useState()
+  const [active, setActive] = React.useState(0)
+  const [newvalue, setNewValue] = React.useState([0, 20])
+  console.log(active)
+  return (
+    <div>
+      <SearchPage
+        portalContent={portalContent}
+        renderSortBy={() => (
+          <>
+            <div className="col">
+              <b>
+                100 Datasets found for "<span>MRI</span>"
+              </b>
+            </div>
+            <div className="col">
+              <div className="search-sort">
+                <SearchSortContainerExample items={sortBy} />
+              </div>
+            </div>
+          </>
+        )}
+        renderSearchFacets={() => (
+          <>
+            <KeywordInputContainerExample searchValue={'MRI'} />
+            <FiltersBlock />
+            <FacetBlockContainerExample>
+              <FacetSelect
```

Comment:
This is expecting a dropdown prop, but `dropdown=undefined` works fine in `FacetSelect` so let's make that optional.